code/helloworld.py

```diff
@@ -81,4 +85,10 @@ def hello(inputfile="data/names.txt", outputfile="results/greetings.txt", sleept
 
     parsed_args = parser.parse_args(args)
 
+    logging.info(
+        "Running helloworld with the following parameters:\n"
```

Comment:
Perhaps sufficient to print `Parameters: inputfile=... outputfile=... sleeptime=...` fitting all in one line?  (See also format comment above.)